app/views/assessments/new.html.erb

```diff
@@ -24,10 +24,10 @@
 
             <ul>
               <% @need.errors.full_messages.each do |message| %>
-                <li><%= message %></li>
+                <li>Assessment <%= message.downcase %></li>
```

Comment:
It was confusing on the errors which field it was referring to, e.g. it would just say "Body is required" rather than "Note body is required"